samtranslator/model/sam_resources.py

```diff
@@ -475,6 +475,9 @@ def to_cloudformation(self, **kwargs):
         """
         resources = []
 
+        intrinsics_resolver = kwargs["intrinsics_resolver"]
```

Comment:
Do we have full test coverage on the different intrinsics supported?

Reply:
The intrinsics I'm resolving (see next line) are just parameter references, which I included in the tests.